packages/service-clients/odsp-client/src/odspAudience.ts

```diff
@@ -10,14 +10,14 @@ import { type OdspMember, OdspUser } from "./interfaces";
 export function createOdspAudienceMember(audienceMember: IClient): OdspMember {
 	const user = audienceMember.user as OdspUser;
 	assert(
-		user.name !== undefined || user.email !== undefined,
+		user.name !== undefined || user.id !== undefined || user.oid !== undefined,
 		0x836 /* Provided user was not an "OdspUser". */,
 	);
 
 	return {
-		userId: user.id,
+		userId: user.oid,
 		name: user.name,
-		email: user.email,
+		email: user.id,
```

Comment:
```suggestion
		email: user.email,
```

Reply:
just don't use IUser.id, it makes the code confusing as its is not what would be expected, and nothing forces its usage since it is redundant via the email property anyway